packaging/windows/variables.wxi

```diff
@@ -26,7 +26,7 @@
   <?if $(var.Platform)=x86?>
   <?define Program_Files="ProgramFilesFolder"?>
   <?define Win64AttributeValue=no?>
-  <?define UpgradeCode="70A1576F-63B6-4659-9E39-25C7B769DDE5"?>
+  <?define UpgradeCode="25CAE344-22A6-422D-942E-942687C56A45"?>
```

Comment:
Installing x86 and x64 side-by-side was the goal (I am open to changing it). I am thinking about the scenario where Visual Studio installs 32-bit CLI into the user machine. At this point can the user install a standalone x64 CLI in their machine? 

We can enable x86 dotnet CLI to cross-compile/build/publish x64 applications and vice versa. In that case only one version of CLI will suffice. But I do not have full details regarding how apps are run on `SharedFramework`. This is definitely a thing in progress. I have created an issue for further discussion - #1570